conf/config.yaml

```diff
@@ -21,3 +21,13 @@
 #     host:
 #       - "http://127.0.0.1:2379"
 #
+#
+# If you want specify the Admin API token, this is an example:
+#
+# apisix:
+#     admin_key:
+#         -
+#             name: "admin"
+#             key:  ******    # <-- replace with a random key
```

Comment:
please keep the same style as https://github.com/apache/apisix/pull/2092/files#diff-4c362b2e3d4cc07f3af3b469be2a913cR75

Reply:
here is a fake example, `******` is an invalid key. 
I think `******` is better than `YOUR_ADMIN_TOKEN`, because `YOUR_ADMIN_TOKEN` is a valid token.

Reply:
how about empty? user don't know `******` is invalid key.

Reply:
will fix it soon